package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "panoptes-client",
-  "version": "4.2.2",
+  "version": "4.2.3",
```

Comment:
Does the `packages` object here also need a version bump like in [176](https://github.com/zooniverse/panoptes-javascript-client/pull/176/files)?

Reply:
oh, good point. Let me check.

Reply:
Should be fixed now.